### PR TITLE
docs: rename templates to suggested names

### PR DESCRIPTION
**Pull request by wtrocki**
See: https://github.com/aerogear/datasync-deployment/pull/3

### CODE REVIEW

**Comment by wtrocki** [comment]
`Data Sync App` name was approved by team.

File: modules/ROOT/pages/_partials/sync/openshift-templates.adoc [hunk 1]
```diff
@@ -6,9 +6,9 @@ that will help developers with provisioning their DataSync applications to OpenS
 
 Templates:
 
-- DataSync API Gateway
+- DataSync App
 
-Provides a way for developers to deploy Node.JS API Gateway application directly from source code. Server can connect with other services running on OpenShift or connect to external data sources like database running on the separate server.
+Provides a way for developers to deploy Node.JS application directly from source code. Server can connect with other services running on OpenShift or connect to external data sources like database running on the separate server.
```
Comment (finp) [inline modules/ROOT/pages/_partials/sync/openshift-templates.adoc]:
```suggestion
Provides a way for developers to deploy Node.js data sync application directly from source code. This server can connect with other services running on OpenShift or connect to external data sources.
```

File: modules/ROOT/pages/_partials/sync/openshift-templates.adoc [hunk 2]
```diff
@@ -6,9 +6,9 @@ that will help developers with provisioning their DataSync applications to OpenS
 
 Templates:
 
-- DataSync API Gateway
+- DataSync App
 
-Provides a way for developers to deploy Node.JS API Gateway application directly from source code. Server can connect with other services running on OpenShift or connect to external data sources like database running on the separate server.
+Provides a way for developers to deploy Node.js data sync application directly from source code. This server can connect with other services running on OpenShift or connect to external data sources.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/sync/openshift-templates.adoc]:
```suggestion
The DataSync App template allows developers to deploy the Node.js DataSync App on Openshift using source code only.  _Node s2i_  is used to build the DataSync App image.

The DataSync App can connect with other services running on OpenShift and can also connect to external data sources.
```

File: modules/ROOT/pages/_partials/sync/openshift-templates.adoc [hunk 3]
```diff
@@ -6,9 +6,9 @@ that will help developers with provisioning their DataSync applications to OpenS
...
 Developers can connect with any services and deploy application directly from source code.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/sync/openshift-templates.adoc]:
```suggestion

```

File: modules/ROOT/pages/_partials/sync/openshift-templates.adoc [hunk 4]
```diff
@@ -6,9 +6,9 @@ that will help developers with provisioning their DataSync applications to OpenS
...
 Template will use Node s2i to build new image.
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/sync/openshift-templates.adoc]:
```suggestion
```

**Comment by wtrocki** [comment]
Good job!

**Pull request merged**